src/reporters/base-reporter.js

```diff
@@ -152,6 +152,8 @@ export default class BaseReporter {
     this.peakMemoryInterval = setInterval(() => {
       this.checkPeakMemory();
     }, 1000);
+    // $FlowFixMe: Node's setInterval returns a Timeout, not a Number
+    this.peakMemoryInterval.unref();
```

Comment:
I'm 👍 with this change but curious why `clearInterval(this.peakMemoryInterval);` in `close()` below isn't enough.